+dj/+internal/GeneralRelvar.m

```diff
@@ -289,7 +289,8 @@ function clip(self)
             for iArg=1:length(specs)
                 % if renamed, use the renamed attribute
                 name = regexp(specs{iArg}, '(\w+)\s*$', 'tokens');
-                if isnumeric(s(1).(name{1}{1})) && length(s(1).(name{1}{1})) == 1
+                sel = cellfun(@(x) strcmp(x, name{1}{1}), {self.header.attributes.name});
+                if self.tableHeader.attributes(sel).isNumeric
```

Comment:
this may have introduced a regression - received a report elsewhere:

```
To fix the bug I changed:
In dj.internal.GeneralRelvar  (line 293) I changed
if self.tableHeader.attributes(sel).isNumeric
to
if self.header.attributes(sel).isNumeric
```

Reply:
looks like this has been filed as an issue in: https://github.com/datajoint/datajoint-matlab/issues/364; disregard discussion here.